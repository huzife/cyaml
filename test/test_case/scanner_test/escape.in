escape: "this\\
 is \t a\n string\'\""
