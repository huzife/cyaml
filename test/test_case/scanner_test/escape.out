(STREAM_START)#
(BLOCK_MAP_START)#
(KEY)#
(SCALAR, escape)#
(VALUE)#
(SCALAR, this\ is 	 a
 string'")#
(BLOCK_MAP_END)#
(STREAM_END)#
